lib/document.js

```diff
@@ -185,6 +191,17 @@ class PDFDocument extends stream.Readable {
     }
   }
 
+  addNamedDestination(name, ...args) {
+    if (args.length === 0) {
+      args = ['XYZ', null, null, null];
+    }
+    if ((args[0] === 'XYZ') && (args[2] !== null)) {
```

Comment:
Why 'XYZ'?

Reply:
XYZ is default option to destination in PDF 1.3 specification, it is means - left (X), top (Y), zoom (Z)

See specification https://www.pdfill.com/download/PDFSPEC13.pdf chapter 7.3.1, page 184